test/e2e/manifest.yaml

```diff
@@ -0,0 +1,30 @@
+# Test manifest for https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/external
+ShortName: ibm-vpc-block
+StorageClass:
+  FromExistingClassName: ibmc-vpc-block-10iops-tier
+SnapshotClass:
+  FromName: true
+DriverInfo:
+  Name: vpc.block.csi.ibm.io
+  SupportedSizeRange:
+    Min: 10Gi
+    Max: 2Ti
+  SupportedFsType:
+    xfs: {}
+    ext4: {}
+  SupportedMountOption:
+    dirsync: {}
+  TopologyKeys: ["failure-domain.beta.kubernetes.io/zone"]
```

Comment:
As a reminder: this key is probably not OK

Reply:
bug to follow up on this: https://bugzilla.redhat.com/show_bug.cgi?id=2035027